docs/source/installation/win.rst

```diff
@@ -92,26 +108,6 @@ Using MiKTex
 2. You can check if you did it right by running ``refreshenv`` to update your
    environment variable and running ``latex``.
 
-Using TinyTex
```

Comment:
So we're no longer using TinyTex at all?

Reply:
Pretty much both are same.

Reply:
@leotrs Is this PR waiting for this?